technicalreport/evaluation.tex

```diff
@@ -13,92 +13,192 @@ \subsubsection{Kriterien}
+
+Für die Vergleichswerte der Verarbeitungszeit und der zusätzlichen Datenmenge wird die Datei \emph{switzerland-exact.osm} \cite{osm_data_switzerland}, Stand 12.11.17, verwendet. Im \ac{OSM}-Format ist die Datei 6.3 GB gross.
+
+
+\begin{figure}[ht]
+    \begin{tikzpicture}
+        \begin{axis}[
+            ybar,
+            width=0.5\textwidth,
+            bar width=14pt,
+            ylabel={Zus. Datenmenge [MB]},
+            ymin=0,
+            ymax=30,
+            symbolic x coords={Visibility,Spiderweb},
+            xtick=data,
+            enlarge x limits=0.6, % pull bars together
+            nodes near coords,
+            nodes near coords align={vertical},
+            every node near coord/.append style={color=black}
+        ] 
+            \addplot[style={orange,fill=orange!30!white}]
+                coordinates {(Visibility,4.1) (Spiderweb,15)};
+        \end{axis}
+    \end{tikzpicture}%
+    ~%
+    % ~ to not break the line 
+    \begin{tikzpicture}
+        \begin{axis}[
+            ybar,
+            width=0.5\textwidth,
+            bar width=14pt,
+            ylabel={Verarbeitungszeit [min]},
+            ylabel near ticks, 
+            yticklabel pos=right, % axis label on the right
+            ymin=0,
+            ymax=60,
+            symbolic x coords={Visibility,Spiderweb},
+            xtick=data,
+            enlarge x limits=0.6, % pull bars together
+            nodes near coords,
+            nodes near coords align={vertical},
+            every node near coord/.append style={color=black},
+        ] 
+            \addplot[style={purple,fill=purple!30!white}]
+                coordinates {(Visibility,13.78) (Spiderweb,32.88)};
+        \end{axis}
+    \end{tikzpicture}
+
+    \caption[Diagramm Datenmenge / Verarbeitungszeit]{Vergleich der zusätzlichen Datenmenge und der Verarbeitungszeit von Visibility Graph und Spiderweb Graph}
+    \label{chart:datenmenge_laufzeit_vergleich}
+\end{figure}
+    
 
 \paragraph{Verarbeitungszeit}\label{result:Verarbeitungszeit}~\\
-TODO Tabelle mit Vergleich der Algorithmen
-\begin{table}[ht]
+Die Vorverarbeitung wurde mit beiden Algorithmen auf der Test-Datei ausgeführt und dabei die benötigte Zeit gemessen. Die Zeit für den ersten Schritt, das Importieren der \ac{OSM}-Daten, wird dabei abgezogen, da dies unabhängig vom jeweiligen Algorithmus ist. Die Resultate sind in Tabelle \ref{Resultat: Verarbeitungszeit} dargestellt.
+
+\begin{table}[H]
     \centering
-    \caption{Resultat: Verarbeitungszeit}
+    \caption{Resultat: Verarbeitungszeit (ohne Initialer Import der Daten)}
     \label{Resultat: Verarbeitungszeit}
     \begin{tabular}{lll}
-        & \textbf{Visiblity Graph} & \textbf{Spiderweb Graph} \\
-        \textbf{Verarbeitungszeit} & x min                    & y min                   
+        & \textbf{Visiblity Graph} & \textbf{Spiderweb Graph}     \\
+        Verarbeitungszeit  &        13m47s                   & 32m53s    \\
```

Comment:
13m47s ist das wegem den Charts in diesem Format oder können wir das in unserem System darstellen?

Reply:
Das ist frei gewählt. Was wäre denn besser? `32:47`?

Reply:
13min 42s wäre doch was